ibis/expr/types/geospatial.py

```diff
@@ -75,6 +121,31 @@ def contains(self, right: GeoSpatialValue) -> ir.BooleanValue:
         -------
         BooleanValue
             Whether `self` contains `right`
+
+        Examples
+        --------
+        >>> from ibis.interactive import *
+        >>> import shapely
+        >>> t = ibis.examples.zones.fetch()
+        >>> p = shapely.Point(935996.821,191376.75) # centroid for zone 1
```

Comment:
Checked, we don't seem to have that API.  But it looks like we can do:

```python
point = ibis.literal((1, 0), type="point").name("p")
point_geom = ibis.literal((1, 0), type="point:geometry").name("p")
```

That being said I'd rather leave the shapely api here, because in `the-epic-split` we can pass that directly.